<launch>
    <!-- <arg name="robot_name" default="small_scout_1" /> -->

    <arg name="output" default="screen" />
    <arg name="use_cheat_odom" default="true" />
    <arg name="use_crab_drive" default="false" />

    <!-- /map /odom 10 number is directly related to the robot_localization frequency! -->
    <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 /map /odom 10" unless="$(arg use_cheat_odom)"/>

    <include file="$(find state_machines)/launch/start_scout.launch">
        <arg name="robot_name" value="small_scout_1" />
        <arg name="output" value="$(arg output)" />
        <arg name="use_cheat_odom" value="$(arg use_cheat_odom)" />
        <arg name="use_crab_drive" value="$(arg use_crab_drive)" />
    </include>

    <include file="$(find state_machines)/launch/start_excavator.launch">
        <arg name="robot_name" value="small_excavator_1" />
        <arg name="output" value="$(arg output)" />
        <arg name="use_cheat_odom" value="$(arg use_cheat_odom)" />
        <arg name="use_crab_drive" value="$(arg use_crab_drive)" />
    </include>

    <include file="$(find state_machines)/launch/start_hauler.launch">
        <arg name="robot_name" value="small_hauler_1" />
        <arg name="output" value="$(arg output)" />
        <arg name="use_cheat_odom" value="$(arg use_cheat_odom)" />
        <arg name="use_crab_drive" value="$(arg use_crab_drive)" />
    </include>

    <!-- <node name="scheduler" pkg="team_level" type="scheduler" args="$(arg robot_name)" output="screen" /> -->
</launch>